documentation/builders/webapp/playlists-livestreams-podcasts.md

```diff
@@ -0,0 +1,141 @@
+# Playlists, Livestreams and Podcasts
+
+By default, the Phoniebox represents music based on its metadata like album name, artist or song name. The hierarchy and order of songs is determined by their original definition. If you prefer a specific list of songs to be played, you can use playlists (files ending with `*.m3u`). Phoniebox also supports livestreams and podcasts (if connected to the internet) through playlists.
+
+## Playlists
+If you like the Phoniebox to play songs in the pre-defined order, you can use .m3u playlists.
+
+An .m3u playlist is a plain text file that contains a list of file paths or URLs to multimedia files. Each entry in the playlist represents a single song, and they are listed in the order in which they should be played.
+
+### Structure of an .m3u playlist
+An .m3u playlist is a simple text document with each song file listed on a separate line. Each entry is optionally preceded by a comment line that starts with a '#' symbol. The actual file paths or URLs of the media files come after the comment.
+
+### Creating an .m3u playlist
+
+1. You can create an .m3u playlist using a plain text editor.
+2. Open a new text file
```

Comment:
You could just use `1.` for all entries, to make future changes easier.
Its the [markdown default](https://docs.github.com/de/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists) and gets replaces on rendering.